docs/Build.md

```diff
@@ -1,10 +1,10 @@
 # Build
 
-This documentation will cover both the [building of the UI code](#ui-build), and also how it is [built and integrated with Strimzi](#ui-build-into-strimzi).
+This documentation will cover both the [building of the UI client and server code](#ui-client-and-server-build), and also how the output of these builds is [built and integrated with Strimzi](#ui-build-into-strimzi).
 
-## UI build
+## UI client and server build
 
-This UI is built using two tools, [Babel](https://babeljs.io/) and [Webpack](https://webpack.js.org/). Webpack acts as our main build and bundling tool, while Babel acts as a transpiler - meaning the UI codebase can make use of the latest and greatest ECMAscript syntax, and Babel will polyfill where appropriate to provide cross browser support. The below will detail choices we have made regarding how the build works, configuration and considerations to be aware of, and the end output. The aim of this stack is to have a fast and efficient build for day to day development, but also the smallest possible built bundles so users do not need to wait a long time for all required assets to be retrieved by their browser.
+This UI codebase (client and server) is built using two tools, [Babel](https://babeljs.io/) and [Webpack](https://webpack.js.org/). Webpack acts as our main build and bundling tool, while Babel acts as a transpiler - meaning the UI codebase can make use of the latest and greatest ECMAscript syntax, and Babel will polyfill where appropriate to provide cross browser support. The below will detail choices we have made regarding how the build works, configuration and considerations to be aware of, and the end output. The aim of this stack is to have a fast and efficient build for day to day development, but also the smallest possible built bundles so users do not need to wait a long time for all required assets to be retrieved by their browser.
```

Comment:
If we go with typescript for the server part as we can use the typescript transpiler instead of Babel if we want.

Reply:
Interesting - can you link me to any more details?

Reply:
Babel added typescript support in Babel 7.  We don't have to use babel the typescript transpiler takes the place of babel if we want.   They do the same thing mostly, with babel missing a few features (e.g. barrel file support isn't as good).  The typescript one is a little ahead of Babel since babel has to add the features after it's released in the typescript transpiler.  I would recommend going with Typescript transpiler instead of Babel so we don't hit the few issues we have to figure out a way around.  Here's a link (https://www.typescriptlang.org/docs/handbook/babel-with-typescript.html).  Like everything it depends on the project.  If we want to go with Babel that's fine with me as well.